apps/admin_api/test/support/conn_case.ex

```diff
@@ -147,11 +147,11 @@ defmodule AdminAPI.ConnCase do
 
     config_pid = start_config_server(account, blockchain_wallet)
 
-    {:ok, keychain_hd_wallet_uuid} = Wallet.generate_hd()
+    {:ok, keychain_hd_wallet} = Wallet.generate_hd()
 
     {:ok, _} =
       BlockchainHDWallet.insert(%{
-        keychain_uuid: keychain_hd_wallet_uuid,
+        keychain_id: keychain_hd_wallet.wallet_id,
```

Comment:
The `uuid` was used to avoid leaking any information about the HD Wallet (pkey, public key, address) and keep it all encapsulated in the Keychain app (for security reasons).